### DISCUSSION TITLE
is a 4th probe possible?

### DISCUSSION THREAD

**Discussion by djbetterly** [artisan-roaster-scope/artisan#1713] [Q&A]
I'm using the Phidget 1048, I see that there are four ports available within Artisan, but not a fourth, is it possible to add a fourth probe?  I have a space for an exhaust probe and I'd love to use it. 

1048 4xTC 01
1048 4xTC 23
1048 4xTC AT

**Comment by MAKOMO** [comment]
The Phidget 1048 has 4 thermocouple inputs (cf. the [1048 module specification](https://phidgets.com/?prodid=1222)). Artisan can read all of those 4 signals. Artisan can also read the on-board module temperature which is used to "calibrate" the 4 TCs.

**Reply by djbetterly** [reply]
okay, so the next question is, how do I read the fourth, and also, how do I calibrate the TC's?  Right now I just have offsets in place to match them to the room temp.

**Reply by djbetterly** [reply]
I asume ET/BT use 01, and for Inlet and Exhaust I'd use 23, but when I select 23 for the exhaust I don't get a reading, I see u.u instead of a reading. But I do see a reading for Inlet temp.

**Comment by djbetterly** [comment]
Any further recommendations on this?

**Comment by roasterdave** [comment]
A little patience would be appreciated before cross posting. We are all volunteers here.  From your other post it sounds like you created a second extra device for exhaust so that you now have two extra devices named "Phidget 1048 4xTC 23".  That won't work.  Delete one of those extra devices and assign both Inlet and Exhaust to the remaining one.  If I misunderstood your configuration then there is either a problem with the probe, wire connection to the Phidget, or the Phidget itself.  

Regarding calibration, from the link Marko provided above please read the User Guide section "Cold Junction Compensation and Self-heating".  There is useful information there, especially "The thermocouple reading is automatically calculated using the data from the on board temperature sensor."

**Reply by djbetterly** [reply]
Apologies, I didn't realize both were monitored by you guys.

Got it..I see what you mean now. I completely overlooked the second LCD part.

I appreciate your help.  I'm sorry for being impatient, have a limited window of time to play with this today.

**Reply by roasterdave** [reply]
Glad it is solved.

**Reply by CarefreeBuzzBuzz** [reply]
> Apologies, I didn't realize both were monitored by you guys.
> 
> 
> 
> Got it..I see what you mean now. I completely overlooked the second LCD part.
> 
> 
> 
> I appreciate your help.  I'm sorry for being impatient, have a limited window of time to play with this today.

I added a sentence to the Quick Start Guide Setup page to clarify.  Can you please update HB to indicate you got this solved on the discussion board or even delete the thread if you prefer.  Thank you.

**Comment by djbetterly** [comment]
Side question, within the device phidgets setup, with the 1048, what is "async" and where can I read about it?

**Reply by roasterdave** [reply]
[On Idle Noise](https://artisan-roasterscope.blogspot.com/2019/03/on-idle-noise.html)  Search the page for "asynchronously".

**Reply by djbetterly** [reply]
Thank you.  I replaced my probes and I feel like something is off since I've done that...trying to work through it.
